pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.0.6</version>
+    <version>2.2.0</version>
```

Comment:
Please change this to 2.3.0 so we can also test out the changes just released the parent around releasing artifacts to Sonatype.